src/tasks/copy-template-files.ts

```diff
@@ -96,7 +100,14 @@ const copyExtensionsFiles = async ({ dev: isDev }: Options, extensionPath: strin
         const isArgs = isArgsRegex.test(path);
         const isTemplate = isTemplateRegex.test(path);
         const isPackageJson = isPackageJsonRegex.test(path);
-        const shouldSkip = isArgs || isTemplate || isPackageJson;
+
+        const unselectedSolidityFrameworks = [SOLIDITY_FRAMEWORKS.FOUNDRY, SOLIDITY_FRAMEWORKS.HARDHAT].filter(
+          sf => sf !== solidityFramework,
+        );
+        const isUselectedSolidityFrameworksRegexes = unselectedSolidityFrameworks.map(sf => new RegExp(`${sf}$`));
+        const isUnselectedSolidityFramework = isUselectedSolidityFrameworksRegexes.some(sfregex => sfregex.test(path));
+
+        const shouldSkip = isArgs || isTemplate || isPackageJson || isUnselectedSolidityFramework;
```

Comment:
The empty (unselected) folder is still being created for me (with no files tho!)

![image](https://github.com/user-attachments/assets/ae8493b7-ed2a-4a68-8977-d7c7862ef8f3)

Reply:
It worked ok for me, no empty folder created.

Reply:
[f5155f5](https://github.com/scaffold-eth/create-eth/pull/90/commits/f5155f54d7dcafb855a83c59fdd158fc602bd719) fixed, in dev mode `linkRecursive` function created folders without checking filters

Reply:
Working great now, thanks for the fix!!